src/pages/FavoriteDetails/FavoriteDetails.page.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Layout from '../../components/Layout';
+import { useParams } from 'react-router-dom';
+import VideoDetail from '../../components/VideoDetail';
+
+function FavoriteDetails({ filter }) {
+  const param = useParams('id');
+
+  return (
+    <Layout>
+      <h2>Favorite Details</h2>
+      <VideoDetail videoId={param.id} videoName="name" favorite="true" />
```

Comment:
`favorite` attribute could be of boolean type so that the internal handling were easier

```suggestion
      <VideoDetail videoId={param.id} videoName="name" favorite={true} />
```

Reply:
or even better:
```suggestion
      <VideoDetail videoId={param.id} videoName="name" favorite />
```
The presence of the attribute makes it to have a truthy value